lib/superset/dashboard/export.rb

```diff
@@ -23,12 +23,35 @@ def initialize(dashboard_id: , destination_path: )
       end
 
       def perform
+        puts "Starting export for dashboard ID: #{@dashboard_id}"
         create_tmp_dir
+        puts "Temporary directory created at: #{tmp_uniq_dashboard_path}"
+
         save_exported_zip_file
+        puts "Exported zip file saved at: #{zip_file_name}"
+
         unzip_files
+        puts "Files unzipped to: #{tmp_uniq_dashboard_path}"
+
+        clean_destination_directory
+        puts "Destination directory cleaned: #{destination_path_with_dash_id}"
+
         copy_export_files_to_destination_path if destination_path
+        puts "Files copied to destination: #{destination_path_with_dash_id}"
+
+        puts "Export process completed successfully."
+
+        # Return the list of copied files
+        Dir.glob("#{destination_path_with_dash_id}/**/*").select { |f| File.file?(f) }
+      rescue StandardError => e
+        puts "Export failed: #{e.message}"
+        raise
+      ensure
+        cleanup_temp_dir
       end
```

Comment:
@barrynguyen8 
please add a logger and move all these puts .. in a logger msg 
similar to this
https://github.com/rdytech/superset-client/blob/develop/lib/superset/dataset/warm_up_cache.rb#L36C5-L38C10

Reply:
@jbat added https://github.com/rdytech/superset-client/pull/40/commits/0dc30db5d997209a4a3167aa9735aa5963278c3b